README.md

```diff
@@ -11,29 +11,223 @@ For more detailed information, visit the [Timeseer.AI website](https://www.times
 - Ansible 2.9 or higher.
 - docker installed on the target machine.
 - `community.docker` Ansible collection.
+- `containers.podman collection` Ansible collection if you are using podman
 
-## Example Playbook
+
+## Role Implementation Examples
+
+The Timeseer Ansible collection includes four roles.
+
+The `timeseer` role is primarily designed for deploying just the Docker container and is ideal for a limited proof of concept (POC) involving 1-2 users. This setup does not require authentication or a web server and can be run locally. However, for an optimal experience, it is recommended to use a sufficiently powerful machine.
+
+By default, this role creates a directory at `/opt/timeseer`. Within this directory, it also creates `db` and `data` subdirectories for your database and data on your local machine. The default variable definitions can be modified to fit your system requirements. Use the variable `timeseer_dir` to change the default directory.
+
+Timeseer uses TOML files for configuration, and the container includes a default configuration file. However, you will likely need to add your own configuration files. You can use the `timeseer_config_dir` variable to define a directory where you can add TOML files, which will be included in the configuration.
+
+Here is an example source file. You can define a source using TOML. For more information on how to set up sources, please refer to our documentation.
+
+
+
+```toml
```

Comment:
Where should I, as a user, store this file?
Do I need to create it on the system?
Do I need to configure it through Ansible (I prefer this)?